packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-rds-walkthrough.ts

```diff
@@ -1,5 +1,5 @@
 import { $TSContext, $TSObject, exitOnNextTick, ResourceCredentialsNotFoundError, ResourceDoesNotExistError } from 'amplify-cli-core';
-import { printer } from 'amplify-prompts';
+import { printer, prompter } from 'amplify-prompts';
 import chalk from 'chalk';
 import { DataApiParams } from 'graphql-relational-schema-transformer';
 import inquirer from 'inquirer';
```

Comment:
Do you still need this in the file?

Reply:
yes - it used on line #246 https://github.com/aws-amplify/amplify-cli/pull/10122/files#diff-6e8be270be62c47a5ef68d95acd1d985d857cc77917d2d391517c288a5fc9eafR246

Reply:
removed :)